### PR TITLE
Add MS teams action

### PR DESCRIPTION
**Pull request by znicholasbrown**
PR Checklist:

- [x] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
More improvements to automations and adds the MS Teams action

Adds more clarity and improvements on top of #749

### CODE REVIEW

File: src/pages/Dashboard/Automations/AddAction.vue [hunk 1]
```diff
@@ -320,19 +334,33 @@ export default {
...
+                  api_token_secret: this.apiToken,
+                  routing_key: this.routingKey,
+                  severity: this.severity
+                }
+                if (this.messageText) {
+                  //TO DO - Don't think this is working - need to fix!
+                  this.actionConfig.message = this.bothMessages
+                    ? `{} ${this.messageText}`
+                    : this.messageText
+                }
```
Comment (whimsicallyson) [inline src/pages/Dashboard/Automations/AddAction.vue:349]: do we want to address this before merging?

Comment (znicholasbrown) [reply src/pages/Dashboard/Automations/AddAction.vue:349]: Oh hm I didn't write this so I'm not sure let me look

**Review by whimsicallyson — DISMISSED** [review]
nice!

**Comment by madkinsz** [comment]
Worked with my teams accnt!

**Pull request merged**